src/measurement/sev_hashes.rs

```diff
@@ -18,6 +20,14 @@ use crate::error::*;
 
 type Sha256Hash = [u8; 32];
 
+#[cfg(not(feature = "openssl"))]
```

Comment:
Similarly here, this wouldn't work when both `openssl` and `crypto_nossl` features are disabled.

```suggestion
#[cfg(all(not(feature = "openssl"), feature = "crypto_nossl")))]
```